app/assets/javascripts/index/new_note.js

```diff
@@ -50,7 +50,8 @@ OSM.NewNote = function (map) {
       data: {
         lat: location.lat,
         lon: location.lng,
-        text: $(form.text).val()
+        text: $(form.text).val(),
+        tags: "created_by:OpenStreetMaps-Website"
```

Comment:
There is no `s` on the end of OpenStreetMap ;-) I'm not sure what is the best value for the tag here but it definitely shouldn't have the trailing s.